backend/app/models.py

```diff
@@ -110,7 +110,7 @@ class Choice(models.Model):
 
 
 class Answer(models.Model):
-  lesson = models.ForeignKey(Lesson, on_delete=models.CASCADE, blank=False)
+  lesson = models.ForeignKey(Lesson, on_delete=models.CASCADE, blank=False, related_name="answers")
```

Comment:
If you're using single-quote, stick to single-quote para consistent. Likewise for double-quotes
```suggestion
  lesson = models.ForeignKey(Lesson, on_delete=models.CASCADE, blank=False, related_name='answers')
```